src/main/java/Deadline.java

```diff
@@ -0,0 +1,39 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    LocalDate taskBy;
```

Comment:
perhaps you might want to consider adding access modifiers to instance variables